src/App.tsx

```diff
@@ -88,7 +91,7 @@ export default class App extends React.Component<IAppProps> {
                                 </TitleBar>
                                 <div className="app-main">
                                     {
-                                        ApiService.getToken() !== null &&
+                                        this.props.auth.accessToken !== null &&
```

Comment:
utiliser le service.testToken ?

Reply:
I'm not sure if this comparison `!== null` isn't too strict.
Just `!!this.props.auth.accessToken && ...` should be enough.
If you cast empty string `''` to boolean with `!!` it will be false. Same as null or undefined. 
You just want to know if `authToken` is falsy, right? It doesn't matter if it's null, undefined or empty string.